solutions/CF Packmen.mdx

```diff
@@ -0,0 +1,295 @@
+---
+id: cf-packmen
+source: CF
+title: Packmen
+author: Kevin Sheng
+---
+
+(No official editorial)
+
+Binary search on the minimum time it will take for all the Packmen to eat all the pellets.
```

Comment:
Do you mean to add additional newlines between these lines to separate them into paragraphs?

Reply:
I guess?

Reply:
So are you going to add those newlines? Right now, it's rendering as one big paragraph

Reply:
Actually, sorry, I didn't know what you meant by that. I meant to render it as a big paragraph, yet I added newlines for ease of editing.